README.md

```diff
@@ -22,6 +22,19 @@ You can use the following table to learn more about each sample, and find the co
 | [High throughput](high-throughput) | Learn how you can design your smart contracts to process a large volume of transactions. | |
 | [Chaincode](chaincode) | A set of sample smart contracts used by other samples and the tutorials in the Fabric documentation. | [Fabric tutorials](https://hyperledger-fabric.readthedocs.io/en/master/tutorials.html) |
 
+### Asset transfer smart contract series
+
+The asset transfer series provides a series of smart contracts and applications that you can use to create and transfer a generic asset using Hyperledger Fabric. However, each sample is built with different smart contract and application APIs in order to demonstrate different Fabric features. The **Basic** sample provides an introduction on how to write smart contracts and how to interact with a Fabric network using the Fabric SDKs. The **Secured agreement** sample demonstrates how to use more advanced capabilities to develop a more realistic transfer scenario.
+
+|  **Smart Contract** | **Description** | **Tutorial** | **Smart contract languages** | **Application languages** |
+| -----------|------------------------------|----------|---------|---------|
+| [Basic](asset-transfer-basic) | The Basic sample smart contract that allows you create and transfer an asset by putting and getting data from the ledger. This sample is recommended for new Fabric users. | **Coming soon** | Go, JavaScript, Typescript | JavaScript, Typescript |
+| [Ledger queries](asset-transfer-ledger-queries) | The ledger queries sample that demonstrates how to deploy an index with your chaincode and issue rich queries when you are using CouchDB as your state database. | **Coming soon** | Go | **Coming soon** |
+| [Private data](asset-transfer-private-data) | This sample demonstrates the use of private data collections and how the private data hash can be used to verify an agreement before executing the transfer | **Coming soon** | Go | **Coming soon** |
```

Comment:
executing the transfer -> executing a transfer

Reply:
done